db/sg_replicate_cfg_test.go

```diff
@@ -649,16 +649,17 @@ func TestReplicateGroupIDAssignedNodes(t *testing.T) {
 	defer tb.Close()
 	ctx := base.TestCtx(t)
 
+	const numCollections = 1
 	// Set up databases
-	dbDefault, err := NewDatabaseContext(ctx, "default", tb.NoCloseClone(), false, DatabaseContextOptions{GroupID: ""})
+	dbDefault, err := NewDatabaseContext(ctx, "default", tb.NoCloseClone(), false, DatabaseContextOptions{GroupID: "", Scopes: GetScopesOptions(t, tb, 1)})
```

Comment:
Did this test work when running with named collections previously?  If yes, was this databaseContext being set up to use `_default._default`?

I agree it's preferable to have the test use a named collection (via GetScopesOptions) if that's how the suite is running, but it also feels like the amount of boilerplate required to just call NewDatabaseContext (with the addition of GetScopesOptions) is getting a bit cumbersome.

Reply:
Made this a constant here, hopefully to add clarity.